src/frame/FrameUtils.ts

```diff
@@ -144,3 +144,8 @@ export function combineDocumentParts(documentID: string, segmentID: number, docu
 export function encryptedDocumentToBase64(documentID: string, segmentID: number, document: EncryptedDocument) {
     return fromByteArray(combineDocumentParts(documentID, segmentID, document));
 }
+
+/**
+ * Encode the provided bytes as hex
+ */
+export const encodeBytesAsHex = (byteSource: Uint8Array): string => Array.prototype.map.call(byteSource, (byte) => `00${byte.toString(16)}`.slice(-2)).join("");
```

Comment:
I googled this trying to understand it and the big [SO hit](https://stackoverflow.com/questions/34309988/byte-array-to-hex-string-conversion-in-javascript) seems to have problems with this approach? I don't really understand it well enough to know if it's a problem, but thought I'd leave it just in case.

Reply:
I think this code is actually OK because of the `Array.prototype.map.call` and not just doing `Uint8Array.map`, but the `Array.from` syntax looks a lot nicer, so I just ended up implementing it that way.